juriscraper/opinions/united_states/state/vt.py

```diff
@@ -112,3 +113,14 @@ def set_url(
             params["facet_to_date"] = end.strftime("%m/%d/%Y")
 
         self.url = f"{self.base_url}?{urlencode(params)}"
+
+    def extract_from_text(self, scraped_text: str):
+        match = re.search(
+            r"(?P<volume>\d{4}) VT (?P<page>\d+)", scraped_text[:1000]
+        )
+        if match:
+            return {
+                "Citation": {"reporter": "VT", "type": 8, **match.groupdict()}
```

Comment:
what about 

r"(?P<volume>\d{4}) (?P<reporter>VT) (?P<page>\d+)", scraped_text[:1000]

instead so we can just do match.groupdict()

Reply:
Sure, I just updated it @flooie